### DISCUSSION TITLE
Individual searchbar - Empty Table

### DISCUSSION THREAD

**Discussion by tmigueis-belodigital** [filamentphp/filament#9947] [Help]
When the table is empty, the individual searchbars don't appear, but I think it makes sense and is more practical to always appear. I made a change to the index file of the filament tables where I resolved this "problem", but as this view is in the vendor folder, when I make an update, this change will disappear, what can I do to prevent this change from disappearing?

**Comment by leandrocfe** [comment]
If you changed the index file, you can create a file in `resources/views/vendor/filament-tables/index.blade.php`, copy/paste the original code and customize what you want. But take a look https://filamentphp.com/community/danharrin-publishing-views-in-laravel

**Comment by tmigueis-belodigital** [comment]
But if i put the file in vendor directory, it will update when i update filament
